r2-streamer-swift/Toolkit/ZIPArchive/ZIPArchive.swift

```diff
@@ -227,7 +227,8 @@ internal class ZipArchive: Loggable {
     /// - Parameter x: The number of bytes to advance the current offset to.
     internal func seek(_ offset: Int) throws {
         let isCompressed = try informationsOfCurrentFile().isCompressed
-        if isCompressed {
+        // FIXME: https://github.com/readium/r2-shared-swift/issues/98
+        if true || isCompressed {
```

Comment:
it looks like Mickael reverted the change that caused that regression so maybe this is no longer needed?

Reply:
I don't know, the issue is open and the code is replaced with a completely different version